### ISSUE TITLE
Include amount of time to destination and estimated time of arrival

### ISSUE THREAD

**Issue by dgearhart** [mapzen/eraser-map#338]
* **Device name:** samsung Model: SM-G900V
* **Android Version:** 5.0
* **App build number:** master-792
* **What did you expected to happen?** To have distance to destination, estimated time to destination, and estimated time of arrival at destination displayed for the user
* **What happened instead?** Only distance to destination was displayed
* **Steps to reproduce:** Run any route
* **Attach a screenshot**
![screenshot_2016-02-08-11-35-03](https://cloud.githubusercontent.com/assets/7515853/12895895/f9f5799c-ce6c-11e5-9d1b-0b6ae71859a7.png)

**Comment by souperneon** [comment]
@dgearhart I decided to leave the time out (both time remaining and ETA) because it's currently not accurate because traffic and other factors are not included.  I take Google's ETA seriously and make mode decisions based on that. Do you think it will be misleading for the user.
We also might end up in a situation where the time might countdown much faster than the distance or time won't change for a while if the user is stuck in  traffic.

**Comment by dnesbitt61** [comment]
Even with traffic information this is just an estimated time...I think some estimate being shown is better than none.

**Comment by dgearhart** [comment]
I think time remaining will update along with distance remaining
I see ETA update on all nav apps depending on how well I am traversing the path.
It would be nice to see how well the base OSM data time estimates are without traffic

**Comment by gknisely** [comment]
I agree, we need an ETA.  While testing in VA this weekend, the family kept asking when will we get there and I had no idea.  Kids even said look at the phone.

**Comment by souperneon** [comment]
Ok I'm sold! I recommend we start with **time remaining** which will be an extension of the choose mode screen where we show distance and time for each mode. 
@dgearhart I'm not sure we are tracking journey time in the app to see how OSM time estimates are. @mjcunningham is that something we can do?

**Comment by dgearhart** [comment]
@souperneon yes we anonymously track trip time and length

**Comment by souperneon** [comment]
For v1, let's try - https://github.com/mapzen/eraser-map/issues/501
